addon/services/alexandria-documents.js

```diff
@@ -38,11 +39,23 @@ export default class AlexandriaDocumentsService extends Service {
       this.selectedDocuments.length === 0
         ? undefined
         : this.selectedDocuments.map((d) => d.id);
-    this.router.transitionTo(this.router.currentRouteName, {
-      queryParams: {
-        document: docs,
-      },
-    });
+
+    if (macroCondition(isTesting())) {
+      // there is no router for integration tests
+      if (this.router.currentRouteName) {
+        this.router.transitionTo(this.router.currentRouteName, {
+          queryParams: {
+            document: docs,
+          },
+        });
+      }
+    } else {
+      this.router.transitionTo(this.router.currentRouteName, {
+        queryParams: {
+          document: docs,
+        },
+      });
+    }
```

Comment:
I don't like this. I would mock the router service instead. 

```javascript
this.owner.lookup("service:router").transitionTo = (routeName) => {});
```

Reply:
Done

Reply:
If you want to, you could also check that `router.transitionTo` has been called.

Reply:
added where it makes sense